swarm/scheduler/filter.md

```diff
@@ -166,6 +167,18 @@ ouruser/sinatra     v2                  cd70495a1514        35 seconds ago
 ubuntu              14.04               a5a467fddcb8        11 days ago         187.9 MB
 ```
 
+### Use the whitelist filter
+
+The node `whitelist` filter directs the scheduler to only consider a specific
+list of nodes for running containers on. The nodes are specified as the value to
```

Comment:
Maybe "The whitelist is comprised of a list of node keys separated by the `|` character."